app/utilities/schema.py

```diff
@@ -27,10 +29,12 @@
 }
 
 
-def get_schema_path_map_for_language(language_code):
+def get_schema_path_map_for_language(
+    language_code: str, dirs: Optional[Tuple[str, ...]] = None
+) -> Mapping:
     schema_files = []
 
-    for schema_dir in DEFAULT_SCHEMA_DIRS:
+    for schema_dir in dirs or DEFAULT_SCHEMA_DIRS:
         schema_files.extend(glob(f"{schema_dir}/{language_code}/*.json"))
```

Comment:
This section could be simplified if the Path object was used consistently throughout the method instead of being used only in the return statement

Reply:
Also, `schema_files` could be created using a list comprehension here

Reply:
Didn't give it much thought as it was existing, will have a look.

Reply:
Have used a `dict` comprehension for this method.